alerts.js

```diff
@@ -0,0 +1,14 @@
+
+// ruleid:javascript-prompt
+var name = prompt('what is your name');
+// ruleid: javascript-alert
+alert('your name is ' + name);
```

Comment:
:exclamation:**Cycode**: SAST violation: 'Found alert() call'.
 
found alert() call; should this be in production code?
 
Severity: Medium
 <!-- b16fd930-13ca-4e52-899e-7f7a70ad5c90 -->